vsts_ci/win32/continuous-build-win32.yml

```diff
@@ -13,7 +13,7 @@ steps:
       npm i -g iothub-explorer yo generator-azure-iot-edge-module
       az --version
       az extension add --name azure-iot
-      & "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe" -m pip install -U setuptools==52.0.0
+      & "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe" -m pip install -U setuptools==57.0.0
```

Comment:
this used to be needed as workaround for an az cli issue, i think we can remove it now